resources/default.rb

```diff
@@ -80,6 +80,24 @@ def platform_config_prefix
   end
 end
 
+# Validates if each element in an array starts with `/` or is in
+# ALL_CAPS. This is helpful for ensuring that the commands
+# passing into the sudoers resource as they need a full path or a
+# `Cmnd_Alias`. This should help people more easily catch issues
+# where the user requested `tail SOME_ARGS SOME_FILE` where they
+# need to use `/usr/bin/tail SOME_ARGS SOME_FILE`.
+# return [TrueClass, FalseClass]
```

Comment:
Wanna toss YARD syntax on this one

Reply:
sorry I missed this, I can get to it maybe this weekend if you have not already fixed it up.